prov/usnic/src/usdf_eq.c

```diff
@@ -171,11 +171,14 @@ usdf_eq_write_event(struct usdf_eq *eq, uint32_t event,
 }
 
 static ssize_t usdf_eq_readerr(struct fid_eq *feq,
-		struct fi_eq_err_entry *entry, uint64_t flags)
+		struct fi_eq_err_entry *buffer, uint64_t flags)
```

Comment:
`buffer` is a little deceiving as a name. If you just saw that name it would imply it was a `char *`.